src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -39,7 +43,10 @@ const BudgetDetailActivityTabContents = ({ enterpriseUUID, enterpriseFeatures })
   if (!isTopDownAssignmentEnabled || !subsidyAccessPolicy?.isAssignable) {
     return (
       <>
-        {!hasSpentTransactions && (<NoBnEBudgetActivity />)}
```

Comment:
[inform] Note: I believe you're rendering `NoBnEBudgetActivity` in certain cases where it should not be rendered... as is, `NoBnEBudgetActivity` is rendered / visible in production for Enterprise Offer budgets that have no spend.

As a result, the `NoBnEBudgetActivity` component (non-functional, without modal) _may_ be visible to admins as is today.

Keep in mind this page is intended to be backwards compatible with legacy Enterprise Offers; not everything here deals with Learner Credit per se. In the case of an enterprise offer used as the specific displayed budget, we should _only_ be rendering the `BudgetDetailRedemptions` component ("Spent" table) in this UI for Learner Credit compatible budgets (_not_ legacy enterprise offers).

As is, I believe `NoBnEBudgetActivity` is rendered for production customers who have an auto-applied budget but no spend (e.g., a newly onboarded customer to an auto-applied budget).

Is the intent for this work to be behind a feature flag until it's complete / fully integrated, so it doesn't expose partially functional components in the UI to production customers?

Reply:
[update] Discussed offline in Slack. The gating of `NoBnEBudgetActivity` behind feature flag will be split out to separate PR to more immediately resolve the production issue where this WIP component is exposed to customers for auto-applied budgets (Learner Credit, or legacy Enterprise Offers) without any spent transactions/redemptions.